src/index.tsx

```diff
@@ -598,28 +606,28 @@ const KEYS = {
 
 const SHAPES = [
   {
-    label: "Rectange",
+    icon: faMousePointer,
+    value: "selection"
+  },
+  {
+    icon: faSquare,
     value: "rectangle"
   },
   {
-    label: "Ellipse",
+    icon: faCircle,
     value: "ellipse"
   },
   {
-    label: "Arrow",
+    icon: faExpandAlt,
```

Comment:
Is there a better icon for an arrow? That doesn't really look like an arrow

Reply:
The Zwibbler uses: https://fontawesome.com/icons/chart-line?style=solid but we have another arrows: https://fontawesome.com/icons?d=gallery&q=arrow&m=free

Reply:
Maybe long-arrow-alt-right